src/button-card.ts

```diff
@@ -508,6 +525,16 @@ class ButtonCard extends LitElement {
         <div id=${key} class="ellipsis" style=${styleMap(customStyle)}>${unsafeHTML(fields[key])}</div>`;
       }
     });
+    Object.keys(cards).forEach((key) => {
+      if (cards[key] != undefined) {
+        const customStyle: StyleInfo = {
+          ...this._buildCustomStyleGeneric(state, configState, key),
+          'grid-area': key,
+        };
+        result = html`${result}
+        <div id=${key} class="ellipsis" style=${styleMap(customStyle)}>${createThing(cards[key], this.hass)}</div>`;
```

Comment:
it's happening!!!!!! :tada:

Reply:
It's not working great for now because of how events are handled :D Still need to fix some stuff